common-api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -196,6 +196,9 @@ public static WorkbenchConfig createEmptyConfig() {
     // Flag to indicate whether to enable the new Create Account flow
     // https://precisionmedicineinitiative.atlassian.net/browse/RW-3284
     public boolean enableNewAccountCreation;
+    // Setting this to true will prevent users from making compute increasing operations on
```

Comment:
If you have a few minutes to write a small markdown file about this mechanism and how it works and when you should make your APIs require billing, you could simply put the URL to that here. I don't think there's enough context here to help folks who come later really understand it.